manual/installation.tex

```diff
@@ -87,16 +87,16 @@ \section{Prerequisites}
 Section \ref{sec:buildperformance} for performance suggestions.
 
 \begin{itemize}
-\item C/C++ compilers such as GCC, Intel, IBM XLC. LLVM-based compilers
-  are supported only on a preliminary basis.
+\item C/C++ compilers such as GNU, Intel, IBM XL. LLVM-based compilers
```

Comment:
Is the statement about LLVM-based compilers still true?  Or do we need to have clang in the regular testing to say it's supported?

Reply:
It is probably no more true. I'm using Clang 4.0 on BGQ and my laptop without any problem.